# This message contains one-epoch measurements from multiple satellites

GnssObsMsg[] meas